lib/testUtils/testFunctions.ts

```diff
@@ -0,0 +1,34 @@
+import { screen, fireEvent, act, Matcher } from "@testing-library/react";
+
+export function expectCallWithoutRejection(toCall: any, reject: any) {
```

Comment:
any

Reply:
replaced with ()=>void but it is possible that if this function will be used more in the future something like ()=>any will be the only way

Reply:
@thatsddr Since you know better what the code is doing than I do, it's okay to leave them in the code as long as u do it consciously. I'm just pointing them out, so you check again with yourself if you placed the any because to keep development pace up and fix it later or because you did it on purpose. 😉